vendor/models/payment.py

```diff
@@ -32,6 +33,7 @@ class Payment(SoftDeleteModelBase):
     success = models.BooleanField(_("Successful"), default=False)
     payee_full_name = models.CharField(_("Name on Card"), max_length=50)
     payee_company = models.CharField(_("Company"), max_length=50, blank=True, null=True)
+    status = models.IntegerField(_("Status"), choices=PurchaseStatus.choices, default=0)
```

Comment:
PurchaseStatus doesn't appear to have a choice with the value 0- maybe this default should be 1?

Reply:
No it is on purpose :)